pages/docs/reference/js-overview.md

```diff
@@ -37,11 +37,26 @@ with a strongly-typed API, you can convert TypeScript definitions from the [Defi
 type definitions repository to Kotlin using the [dukat](https://github.com/kotlin/dukat) tool. Alternatively, you can use
 the [dynamic type](dynamic-type.html) to access any framework without strong typing.
 
-JetBrains develops and maintains several tools specifically for the React community: [React bindings](https://github.com/JetBrains/kotlin-wrappers) as well as [Create React Kotlin App](https://github.com/JetBrains/create-react-kotlin-app). The latter helps you start building React apps with Kotlin with no build configuration.
-
 Kotlin is compatible with CommonJS, AMD and UMD, [making interaction with different](/docs/tutorials/javascript/working-with-modules/working-with-modules.html) module systems straightforward.
 
 
-## Getting Started with Kotlin to JavaScript
+## Getting Started with Kotlin/JS
+
+To find out how to start using Kotlin for JavaScript, please refer to the [Setting up a Kotlin/JS project](/docs/reference/js-project-setup.html).
```

Comment:
This setting up seems too long and complex for getting started. We talked with Seb that we should shorten it in the future. 
But since there is no any other option for no, let's use this link. Setting up in tutorials seems abrupt.